webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationLayer.java

```diff
@@ -71,6 +71,8 @@
     private boolean builtIn = false;
     
     private boolean readonly = false;
+    
+    private String onClickJavascriptAction;
```

Comment:
Since the new property will be unset for any entities already in the database, it must be nullable.